Sources/UXKit/AppKit/UXGraphics-AppKit.swift

```diff
@@ -65,5 +67,76 @@
     static var applicationIconImage: UXImage? {
       return UXImage(named: NSImage.applicationIconName)
     }
+    
+    /// Returns an image of the specified size, allowing the caller to provide a draw function that draws into the "current" graphics context.
+    /// - Parameters:
+    ///   - size: The size of the image
+    ///   - drawContent: A block responsible for drawing into the image.
+    /// - Returns: The image (which may be blank).
+    static func image(withSize size: CGSize, andContent drawContent: () -> Void) -> UXImage {
+        let resultImage = NSImage(size: size)
+        if let rep = NSBitmapImageRep(bitmapDataPlanes: nil, pixelsWide: Int(size.width), pixelsHigh: Int(size.height), bitsPerSample: 8, samplesPerPixel: 4, hasAlpha: true, isPlanar: false, colorSpaceName: .calibratedRGB, bytesPerRow: 0, bitsPerPixel: 0) {
+            resultImage.addRepresentation(rep)
+            NSGraphicsContext.saveGraphicsState()
+            NSGraphicsContext.current = NSGraphicsContext(bitmapImageRep: rep)
+            
+            drawContent()
+            
+            NSGraphicsContext.restoreGraphicsState()
+        }
+            
+        return resultImage
+    }
   }
+
+import class SpriteKit.SKScene
+
+public extension UXTouch {
+    
+    func location(inScene: SKScene) -> UXPoint {
+        return self.location(in: inScene.view)
+    }
+}
+
+public extension NSBezierPath {
+    
+    /// CREDIT:  https://gist.github.com/juliensagot/9749c3a1df28c38fb9f9
+    /// A `CGPath` object representing the current `NSBezierPath`.
+    var cgPath: CGPath {
+        let path = CGMutablePath()
+        let points = UnsafeMutablePointer<NSPoint>.allocate(capacity: 3)
+
+        if elementCount > 0 {
+            for index in 0..<elementCount {
+                let pathType = element(at: index, associatedPoints: points)
+
+                switch pathType {
+                case .moveTo:
+                    path.move(to: points[0])
+                case .lineTo:
+                    path.addLine(to: points[0])
+                case .curveTo:
+                    path.addCurve(to: points[2], control1: points[0], control2: points[1])
+                case .closePath:
+                    path.closeSubpath()
+                    case .cubicCurveTo:
```

Comment:
This is completely misaligned and doesn't compile w/ Xcode 14.2. I think to get this to work, we'd need `#if swift(5.x.x)` checks for both code pathes.
Also macOS 14 now supports this? We should probably use it if available. I.e. this probably needs to be converted to an `uxCGPath`, call cgPath on iOS and macOS 14, and then do the implementation as required for older macOS.